tests/runtest.cmd

```diff
@@ -2,9 +2,11 @@
 setlocal EnableDelayedExpansion
 set __ProjectFilesDir=%~dp0
 
-:: Default to VS2013
-set __VSVersion=VS2013
-set __VSProductVersion=120
+:: Default to highest Visual Studio version available
+set __VSVersion=vs2015
```

Comment:
The usage message below still says that the default is 2013.  Can you change that?